x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/search_application_layout.scss

```diff
@@ -0,0 +1,5 @@
+
+.searchApplciationHeaderBackgroundColor {
+  background-color:  rgba(247, 248, 252, 1);
```

Comment:
Can this be an variable from EUI? cc @daveyholler

Reply:
You can use `$euiTextSubduedColor`

Reply:
You should be able to use `$euiColorLightShade` for this. The color isn't actually called `subdued`, but the light shade is the background color for a subdued panel.

Reply:
I think` $euiColorLightestShade` is closest to `subdued` as `$euiColorLightShade` is little more of dark.

Reply:
Page with background color ` $euiColorLightestShade` 

<img width="1728" alt="Screenshot 2023-04-26 at 9 39 39 AM" src="https://user-images.githubusercontent.com/55930906/234593587-e80aa4bd-d196-4a2e-9bb6-b5302e31c47d.png">

cc: @julianrosado WDYT?

Reply:
🤩 Beautiful!